plugin/src/main/java/io/spine/tools/gradle/bootstrap/SpinePluginScripts.java

```diff
@@ -64,7 +64,7 @@ static PluginScript modelCompilerConfig() {
 
         private Resource resourceFile() {
             String resourceName = name + SCRIPT_EXTENSION;
-            return Resource.file(resourceName);
+            return Resource.file(resourceName, SpinePluginScripts.class.getClassLoader());
```

Comment:
Please consult with @dmdashenkov on this matter. We have encountered the similar issues in our plugin codebase already. I am worrying that you might be overriding the previous "fix" here.

Reply:
This looks OK. The method with a single argument is now deprecated. From a discussion with @yuri-sergiichuk, this is the reason for the substitution, not a runtime error.